web/wp-content/plugins/tmbi-adobe-dtm/inc/adobe_dtm_utils.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/*
+ * User: jeysaravana
+ * Date: 2018-01-29
+ * Time: 12:18 PM
+ */
+
+class Adobe_DTM_Utils {
```

Comment:
What's the purpose of this class? Reading through the code it seems this is used to store the original publishing date in post meta. If so, name could be more descriptive.